docs/pages/web/icon.tsx

```diff
@@ -222,6 +223,20 @@ Should be used sparingly and only in places where the UI is very dense and a lar
             title="Custom SVG icon"
           />
         </MainSection.Subsection>
+        <MainSection.Subsection
```

Comment:
Can we do a follow up PR adding a single page for IconCompact?

Reply:
Yes - I think we should. I wanted to wait since there's not too many right now. and it's a replica of `Icon`